lib/smack/IntegerOverflowChecker.cpp

```diff
@@ -119,6 +119,20 @@ bool IntegerOverflowChecker::runOnModule(Module& m) {
     if (Naming::isSmackName(F.getName()))
       continue;
     for (inst_iterator I = inst_begin(F), E = inst_end(F); I != E; ++I) {
+      // Add check for UBSan left shift
+      if (auto chkshft = dyn_cast<CallInst>(&*I)) {
+        if (chkshft->getCalledFunction() != nullptr &&
+            chkshft->getCalledFunction()->getName().find("__ubsan_handle_shift_out_of_bounds") != std::string::npos &&
```

Comment:
I think you have to check here first if function actually has name using `hasName`. Maybe make it similar to this line:
https://github.com/smackers/smack/blob/901550325ca9112fb6d8008b19a9e66976ed169b/lib/smack/IntegerOverflowChecker.cpp#L140

Reply:
Done.